docs/category.json

```diff
@@ -7,35 +7,9 @@
     "user/admin/settings.rst"
   ],
   "ppl_cli": [
-    "user/ppl/cmd/ad.rst",
-    "user/ppl/cmd/dedup.rst",
-    "user/ppl/cmd/describe.rst",
-    "user/ppl/cmd/showdatasources.rst",
-    "user/ppl/cmd/eval.rst",
-    "user/ppl/cmd/fillnull.rst",
     "user/ppl/cmd/grok.rst",
-    "user/ppl/cmd/head.rst",
     "user/ppl/cmd/parse.rst",
-    "user/ppl/cmd/patterns.rst",
-    "user/ppl/cmd/rare.rst",
-    "user/ppl/cmd/search.rst",
-    "user/ppl/cmd/sort.rst",
-    "user/ppl/cmd/syntax.rst",
-    "user/ppl/cmd/trendline.rst",
-    "user/ppl/cmd/top.rst",
-    "user/ppl/cmd/where.rst",
-    "user/ppl/cmd/join.rst",
-    "user/ppl/cmd/lookup.rst",
-    "user/ppl/cmd/subquery.rst",
-    "user/ppl/general/identifiers.rst",
-    "user/ppl/general/datatypes.rst",
-    "user/ppl/functions/datetime.rst",
-    "user/ppl/functions/expressions.rst",
-    "user/ppl/functions/ip.rst",
-    "user/ppl/functions/json.rst",
-    "user/ppl/functions/math.rst",
-    "user/ppl/functions/relevance.rst",
-    "user/ppl/functions/string.rst"
+    "user/ppl/cmd/patterns.rst"
```

Comment:
Only grok, parse, patterns are still not able to 100% match the result of doctest in v2. @songkant-aws can you fix it or link an issue.